File: cakk-api/src/main/java/com/cakk/api/dto/response/user/ProfileInformationResponse.java
package com.cakk.api.dto.response.user;

import java.time.LocalDate;

import lombok.Builder;

import com.cakk.common.enums.Gender;
import com.cakk.common.enums.Role;

@Builder
public record ProfileInformationResponse(
	String profileImageUrl,
	String nickname,
	String email,
	Gender gender,
	LocalDate birthday,
	Role role
) {
}

File: cakk-core/src/main/kotlin/com/cakk/core/dto/response/user/ProfileInformationResponse.kt
package com.cakk.core.dto.response.user

import java.time.LocalDate

import com.cakk.common.enums.Gender
import com.cakk.common.enums.Role

data class ProfileInformationResponse(
    val profileImageUrl: String?,
    val nickname: String,
    val email: String,
    val gender: Gender,
    val birthday: LocalDate,
    val role: Role
)
